evo_researcher/benchmark/benchmark.py

```diff
@@ -318,21 +318,35 @@ def calculate_expected_returns(self, prediction: Prediction, market: Market) ->
 
         # TODO: Add support for different bet sizes and calculate shares based on the market's odds.
         bet_units = 10  # Assuming the agent always bet 10 units per market.
-        receive_shares = 20  # Because we assume markets trades at 50/50.
         buy_yes_threshold = 0.5  # If the agent's prediction is > 50% it should buy "yes", otherwise "no".
 
         assert prediction.outcome_prediction is not None
-        yes_shares = receive_shares if prediction.outcome_prediction.p_yes > buy_yes_threshold else 0
-        no_shares = receive_shares if prediction.outcome_prediction.p_yes <= buy_yes_threshold else 0
-        
-        expected_returns_pct = (
+        # Assume that market starts at 50/50 and so the price is 0.5 at the time we are buying it,
+        # we can't use {yes,no}_outcome_price atm, because it would just cancel out to EV = 0.0,
+        # as it's the same as the probability.
```

Comment:
Just added the comment because I confused myself in the morning 😄

Reply:
That's just confused me too! 😄 Maybe could do with adding a block comment at the start of the function, like:
```
"""
The expected value if betting on a binary market in its initialized state of 50:50 'yes' and 'no' shares, with the assumption that the correct `p_yes` is that of the market.
"""
```

Can also take away the `# TODO` on 319, as:
- the returned expected value is as a %, so the size of `bet_units` doesn't matter.
- as you say, we can't calculate shares based on market odds if we're also using market odds as the 'correct' odds

Reply:
> adding a block comment

Added

> Can also take away the # TODO on 319, as:

I updated it, bet size actually should make a difference, but it doesn't in our simplified calculation atm. But I would leave the comment there, so we don't forget about it.